StyleGuide.md

```diff
@@ -206,124 +322,142 @@ Try to address as many of the suggestions as possible. If Vale flags a word that
 
-#### Notes
+- Use a _warning_ to express that a piece of information is critical to understand to prevent data loss, security vulnerabilities, or unexpected behavior.
 
-Use a note to call attention to a piece of clarifying information; this information should not be crucial to accomplishing the task in the document.
+  For example, you might include a warning that using `CASCADE` in `DROP INDEX` drops dependent objects without warning. This is critical to prevent users from unexpectedly losing constraints or additional indexes.
 
-For example, you might use a note to let users know that the `DELETE` statement only deletes rows and that to delete columns you must use `ALTER TABLE`. This helps clarify `DELETE`‘s purpose and point users to the right place.
+**Best practices:**
 
-To insert a note, use the following code:
+- Avoid placing callouts adjacent to each other.
+- Do not overuse callouts. Most documentation belongs in the body of a page rather than in a callout.
 
-~~~
-{{site.data.alerts.callout_info}}
-<note text goes here>
-{{site.data.alerts.end}}
-~~~
+### Images
+
+Use images to clarify a topic, but only use them as needed. Images are either:
+
+- **Screenshots:** Depict a UI element. Screenshots should show enough of the UI that the user can easily orient themselves and understand what they are being shown. If a screenshot needs an annotation, use a red box.
 
-#### Warnings
+  **Note:** Screenshots are difficult to keep current, and impact accessibility. Use a screenshot only if it is necessary for the user to accomplish a task or understand a feature. For example, in a page on troubleshooting cluster performance, a screenshot of a Contention metric graph in the DB Console might be used to depict a cluster with high contention. The same screenshot would not be necessary in a reference topic about the DB Console user interface. 
 
-Use a warning to express that a piece of information is critical to understand to prevent data loss, security vulnerabilities, or unexpected behavior.
+  To optimize accessibility and SEO, a screenshot should be accompanied by text and [alt-text](#write-accessible-documentation) describing its contents.
 
-For example, you might include a warning that using `CASCADE` in `DROP INDEX` drops dependent objects without warning. This is critical to prevent users from unexpectedly losing constraints or additional indexes.
+- **Diagrams:** Visualize a complicated mechanism. Diagrams should not reproduce such complexity, and can simplify a process. A diagram should be easy to follow.
 
-To insert a warning, use the following code:
+  The Docs team uses the following tools to compose diagrams:
 
-~~~
-{{site.data.alerts.callout_danger}}
-<warning text goes here>
-{{site.data.alerts.end}}
-~~~
+  - [Monodraw](https://monodraw.helftone.com/)
+  - [Omnigraffle](https://www.omnigroup.com/omnigraffle)
```

Comment:
Suggest appending the use cases for each. IMO these are
- Monodraw (useful for abstract concept diagrams)
- Omnigraffle (useful for diagrams with branded elements like logos and colors)